tests/staged/data/simple_errata/dest1/ERRATA/advisory2.json

```diff
@@ -159,6 +159,24 @@
   "pulp_user_metadata": {
     "content_types": [
       "rpm"
+    ],
+    "container_list": [
```

Comment:
The sample data here doesn't seem to be correct as the format is neither the same as what ET produces (as build NVRs were replaced with a repo name), nor what ErratumPushItem.container_list is expected to be (as image archive names have not been replaced with arches).

What format is supposed to be used for staged errata?  It seems that it should be either the ET native format or the ErratumPushItem format and not something in-between.

In either case we should arrange it so that the keys under `images` are expected to be arch names in the staged case, same as it is in the ET case.

Reply:
My understanding was that it should be provided in already transformed format. So images needs to be replaced with architectures